File: DIPLOMADO/COLABORADOR_1/ESTRUCTURA DE DATOS/PROYECTO_EVALUATIVO_FRANCISCO_CAMPOS_SANDI/ArticulosFerreteria/src/articulosferreteria/Articulo.java
/*
UNIVERSIDAD ESTATAL A DISTANCIA
Curso:Estructuras de datos
Código: 0825
Proyecto evaluativo:  Artículos ferretería
Tutor:Steve Brenes Torres
Grupo:03
Estudiante: Francisco Campos Sandi
Cédula: 114750560
II Cuatrimestre 2024
*/


package articulosferreteria;

public class Articulo {
     // Atributos
    private int id;
    private String nombre;
    private double precio;
    // Constructor para inicializar
    public Articulo(int id, String nombre, double precio) {
        this.id = id;
        this.nombre = nombre;
        this.precio = precio;
    }
    // Métodos getter
    public int getId() {
        return id;
    }

    public String getNombre() {
        return nombre;
    }

    public double getPrecio() {
        return precio;
    }

    @Override
    public String toString() {
        return "ID: " + id + ", Nombre: " + nombre + ", Precio: " + precio;
    }
}

File: DIPLOMADO/COLABORADOR_1/AVANZADA/PROYECTO_1/FranciscoCampos_Proyecto_1/TiendaDeportiva/CapaEntidades/Articulo.cs
/*
UNIVERSIDAD ESTATAL A DISTANCIA
Curso: Programación avanzada
Código: 00830
Proyecto #1: Tienda deportiva
Tutor: Juan Ramírez Valladares
Grupo: 09
Estudiante: Francisco Campos Sandi
Cédula: 114750560
III Cuatrimestre 2024
*/
using System;
namespace TiendaDeportiva.CapaEntidades
{
    public class Articulo
    {// atributos de la clase Articulo
        public int Id { get; set; }
        public string Descripcion { get; set; }
        public Categoria CategoriaArticulo { get; set; }
        public string Marca { get; set; }
        public bool Activo { get; set; }
        // Constructor de la clase Articulo para inicializar un nuevo artículo con todos los datos
        public Articulo(int id, string descripcion, Categoria categoria, string marca, bool activo)
        {
            this.Id = id;
            this.Descripcion = descripcion;
            this.CategoriaArticulo = categoria;
            this.Marca = marca;
            this.Activo = activo;
        }
    }

}
